src/Core/SettingsChangesHistory.h

```diff
@@ -114,6 +114,7 @@ static const std::map<ClickHouseVersion, SettingsChangesHistory::SettingsChanges
               {"output_format_csv_serialize_tuple_into_separate_columns", true, true, "A new way of how interpret tuples in CSV format was added."},
               {"input_format_csv_deserialize_separate_columns_into_tuple", true, true, "A new way of how interpret tuples in CSV format was added."},
               {"input_format_csv_try_infer_strings_from_quoted_tuples", true, true, "A new way of how interpret tuples in CSV format was added."},
+              {"output_format_parquet_write_page_index", false, false, "Write page index into parquet files."},
```

Comment:
```suggestion
              {"output_format_parquet_write_page_index", false, false, "Add a posibility to write page index into parquet files."},
```

I think because the settings is not enabled by default, this note in setting changes might be confusing: it suggests that page index is written by default.